fcos-config.yaml

```diff
@@ -91,3 +117,48 @@ storage:
         inline: |
           [engine]
           machine_enabled=true
+    - path: /usr/local/bin/verify-bearer-token
+      mode: 0755
+      contents:
+        inline: |
+          #!/bin/sh
+          # This file is part of CRC
+          # Note: this is a POC and should not be used in production
+          #
+          # Copyright (C) 2021 Red Hat, Inc.
+          set -eux
+          HOST="$1"
+          echo "Bearer auth attempt from $HOST" >&2
+          # FIXME: should be random
```

Comment:
maybe this could be fixed?

Reply:
We need to do that in sync with CRC, as the tray-electron would need to read this bearer and use for authentication. Let's start with this as part of the test and then implement a file or http API to retrieve the needed bearer.

Reply:
https://github.com/code-ready/tray-electron/issues/24#issue-1082456798